tests/api2/test_420_smb.py

```diff
@@ -410,16 +410,18 @@ def get_event(event_list, ev_type):
             assert new_data['audit']['ignore_list'] == ['builtin_users'], str(new_data['audit'])
 
             # Verify that being member of group in ignore list is sufficient to avoid new messages
-            assert len(do_audit_ops(s['name'])) == len(events)
+            # By default authentication attempts are always logged
+            assert len(do_audit_ops(s['name'])) == len(events + 1)
```

Comment:
These are fine, but I do also wonder whether it might make sense to ensure the events look right, rather than just relying upon a count.

Reply:
That's a good idea.